tasks.py

```diff
@@ -39,9 +39,9 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_ssot": {
-            "nautobot_ver": "1.1.4",
+            "nautobot_ver": "1.3.3",
```

Comment:
```suggestion
            "nautobot_ver": "1.3.7",
```

Reply:
Going a step further and including today's 1.3.8 release.